index.html

```diff
@@ -204,19 +209,48 @@ <h4>Languages</h4> <img class="arrow" src="images/Disabled.png" alt="">
                         <li class="language"><img src="images/Ellipse_css.png" alt="">CSS</li>
                     </ul>
                 </li>
-                <li><img class="separator"img src="images/Separator Bottom.png" alt=""></li>
+                <li><img class="separator" img src="images/Separator Bottom.png" alt=""></li>
                 <li>
-                    <div class="frame"><h4>Frameworks</h4> <img class="arrow" src="images/right.png" alt="">
+                    <div class="frame">
+                        <h4>Frameworks</h4> <img class="arrow" src="images/right.png" alt="">
                     </div>
                 </li>
-                <li><img class="separator"img src="images/Separator Bottom.png" alt=""></li>
+                <li><img class="separator" img src="images/Separator Bottom.png" alt=""></li>
                 <li>
-                    <div class="frame"><h4>Skills</h4><img class="arrow" src="images/right.png" alt=""></div>
+                    <div class="frame">
+                        <h4>Skills</h4><img class="arrow" src="images/right.png" alt="">
+                    </div>
                 </li>
             </ul>
         </div>
     </section>
     <!-- end of about section -->
-</body>
 
+    <!-- contact form -->
+    <section class="contact_form">
+        <div class="contact_wrapper">
+            <h1>Contact me</h1>
+            <p>
+                If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding.
+                I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xbjeloeq" method="post">
+                <ul>
+                    <li>
+                      <input name="username" type="text" placeholder="Your name" maxlength="30"required>
+                    </li>
+                    <li>
+                      <input name="email" type="email" placeholder="youremail@example.com" required>
+                    </li>
+                    <li>
+                      <textarea name="message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
+                    </li>
+                </ul>
```

Comment:
- [OPTIONAL] The inputs of a form usually are not list items so you could remove the `ul` and `li` inside this form and it would be semantically correct too.